bundles/org.openhab.binding.vesync/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -242,14 +263,14 @@
 	</channel-type>
 
 	<channel-type id="deviceAFConfigAutoPrefRoomSizeType">
-		<item-type>Number:Dimensionless</item-type>
+		<item-type unitHint="one">Number:Dimensionless</item-type>
```

Comment:
This seems like a channel that should use UoM
```suggestion
		<item-type>Number:Area</item-type>
```

Reply:
This has been updated now, and is mapping correctly.